### ISSUE TITLE
Bring over msmtools

### ISSUE THREAD

**Issue by brookehus** [scikit-time/scikit-time#15]
For the goal of eventually reducing/removing nontrivial dependencies.

This is going to involve merging the `setup.py`'s to build extensions

**Comment by marscher** [comment]
I modernized msmtools to use numpy.distutils. That way it is easy to include it as a **subtree**, because numpy distutils defines packages relatively to their parent instead of absolutely like setuptools does. I also separated the tests from the core package, so it has become cleaner. We can still think about removing unused code (unused by sktime), but after looking at it, it would not shrink the package much further. There are several utilities functions even duplicated right now, and they have diverged slightly over the time, so I guess it would be great to back integrate things to lower the overall maintenance burden.

**Comment by clonker** [comment]
done via PR #74

**Comment by marscher** [comment]
Is there any notice within the old msmtools repo, that it will not be maintained anymore (soon/at the momet)?
Today I was in contact with Bernhard Reuter and he will contribute the generalized PCCA to msmtools very soon. To avoid double effort in the future I strongly suggest to add this notice to msmtools and redirect to this project for now. Thanks a lot!

**Comment by clonker** [comment]
Thanks for the reminder @marscher , I am going to put up a notice! And also port g-pcca to this package once it is merged into msmtools.
